python/tvm/auto_scheduler/auto_schedule.py

```diff
@@ -161,7 +161,7 @@ def __init__(self, task, schedule_cost_model=RandomModel(), params=None, seed=No
             seed or random.randint(1, 1 << 30), verbose, init_search_callbacks)
 
     def generate_sketches(self, print_for_debug=False):
-        """ Generate the sketches, this is mainly used for debug.
+        """ Generate the sketches. This is mainly used for debugging and testing.
```

Comment:
I know the meaning of this description, but it may confuse people. Maybe we can either don't say this is mainly for debugging and testing, or explicitly say this is mainly for debugging and testing because auto scheduler uses them in the C++ side?